packages/botonic-plugin-flow-builder/tests/bot-action.test.ts

```diff
@@ -33,21 +34,28 @@ describe('The user clicks on a button that is connected to a BotActionNode', ()
     expect(nextPaylod).toBe(botActionUuid)
   })
 
-  test('The request.session._botonic_action have redirect:nextPayload', async () => {
-    const { contents, request } = await createFlowBuilderPluginAndGetContents({
-      flowBuilderOptions: { flow: basicFlow },
-      requestArgs: {
-        input: {
-          type: INPUT.POSTBACK,
-          payload: botActionUuid,
-        },
+  test('The request.input.payload after execute pre function of the plugin is nextPayload with params', async () => {
```

Comment:
Suggestion: Maybe the test description could be improved to show a little bit better what is tested? You're only testing here that the payload is changed in the pre execution if a Bot Action UUID is detected in the payload, right?

Reply:
Changed to `When the request.input.payload is a UUID of a bot action node it is replaced by the payload with parameters defined in the node`